XML/Other/ReplaceableTestParameters.xml

```diff
@@ -729,4 +729,16 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>DPDK_FAILSAFE_SH_URL</ReplaceThis>
 		<ReplaceWith>https://gallery.technet.microsoft.com/Azure-DPDK-Failsafe-PMD-c34b1a4b/file/183012/1/failsafe.sh</ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<ReplaceThis>NESTED_IMAGE_URL</ReplaceThis>
+		<ReplaceWith></ReplaceWith>
```

Comment:
According to your design, people who are not familiar with this script inside will get 'TestAborted' for the first time mostly because there is no default value. Is there any other better way to approach?

Reply:
Hi @juhlee-microsoft ,
I don't know if there's a public and stable http server to storage the image. I get the nested image from here:
https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.2.2004-20200611.2.x86_64.qcow2
And use virt-customize to set root password and remove cloud-init. But IMO there are several issues if use this image url as default value:
1. It might be changed in the future. (or we can crawling the url from https://cloud.centos.org/centos/8/x86_64/images/)
2. The origin image doesn't contain root password. We need to install many packages(e.g. libvirt, virt-customize...) to modify it in the guest VM, which may cost more time.

Another way is using my own set http server, which is in a VM on Azure and contains a customized centos 8 image.  

Which one do you think is better? Or any other suggestions? Thanks!

Reply:
Definitely, I prefer to use the public image. If it does not have the password, that is fine. Actually this comes from either @LiliDeng  or @lizzha before. They might have better insight. Can either of you comment back for us? @LiliDeng  @lizzha

Reply:
these value we set in secrets file, not sure if setting here will be conflict with that or not, let me have a try.

Reply:
@LiliDeng Any progress?

Reply:
tested, doesn't break our testing.